src/core/src/main.rs

```diff
@@ -40,6 +40,7 @@ pub mod database;
 pub mod consensus_state;
 pub mod consensus_state_store;
 pub mod poet2_util;
+pub mod poet2_settings_view;
```

Comment:
Do we really need to prefix poet2 for a module? That would only elongate the name.

Reply:
Good point. Let me fix that.